payjoin/src/core/uri/v2.rs

```diff
@@ -301,6 +312,8 @@ impl std::fmt::Display for ParseOhttpKeysParamError {
             MissingOhttpKeys => write!(f, "ohttp keys are missing"),
             InvalidOhttpKeys(o) => write!(f, "invalid ohttp keys: {o}"),
             InvalidFragment(e) => write!(f, "invalid URL fragment: {e}"),
+            InvalidHrp(h) => write!(f, "incorrect hrp for exp: {h}"),
+            DecodeBech32(d) => write!(f, "exp is not valid bech32: {d}"),
```

Comment:
The displayed error message seems to have changed.
Previously it was 
```rust
 ParseOhttpKeysError::DecodeBech32(e) => write!(f, "Failed to decode bech32: {e}"),
```

Was this intentional?

Reply:
I also notice the next commit removes these variants. Can we not just remove them in this commit?